libs/partners/openai/langchain_openai/chat_models/base.py

```diff
@@ -889,6 +916,35 @@ def _create_chat_result(
 
         return ChatResult(generations=generations, llm_output=llm_output)
 
+    def _create_chat_result_responses(
+        self, response: Response, generation_info: Optional[Dict] = None
+    ) -> ChatResult:
+        generations = []
+
+        if error := response.error:
+            raise ValueError(error)
+
+        token_usage = response.usage.model_dump() if response.usage else {}
+        generation_info = {}
+        for output in response.output:
+            if output.type == "message":
+                joined = "".join(
```

Comment:
should we just keep the content blocks so annotations are included? thats what we do for anthropic right

Reply:
or we could check for annotations and join if there aren't any